docs/src/main/asciidoc/spanner.adoc

```diff
@@ -34,7 +34,7 @@ Spring Data Spanner allows you to define entity classes and corresponding reposi
 ----
 @SpannerTable(name = "students")
 public class SimpleStudent {
-	@Id
+	@SpannerPrimaryKeyColumn(keyOrder = 1)
```

Comment:
How about `@PrimaryKeyColumn`?

Reply:
Thank you so much for providing all this feedback! 

For this new annotation I wanted to follow the way we've been doing the other annotations like SpannerTable , SpannerColumn, etc...

Reply:
I agree with `@PrimaryKeyColumn`, see #489